django/db/models/base.py

```diff
@@ -1665,6 +1666,25 @@ def _check_long_column_names(cls):
 
         return errors
 
+    @classmethod
+    def _check_indexes(cls):
+        """
+        Check fields in "indexes" option are valid.
+        """
+        errors = []
+        field_names = [field.name for field in cls._meta.fields]
```

Comment:
on second read, could be a set so the `not in` check below is slightly faster - just swap `[]` for `{}`